data/json/items/gun/shot.json

```diff
@@ -637,9 +635,49 @@
       [ "sling", 1 ]
     ],
     "flags": [ "RELOAD_ONE" ],
-    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "shot": 5 } } ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "shot": 15 } } ],
     "melee_damage": { "bash": 12 }
   },
+  {
+    "id": "dp_12",
+    "copy-from": "shotgun_pump",
+    "type": "GUN",
+    "name": { "str": "two-barrel pump shotgun" },
+    "description": "Hefty shotgun, that has two barrels, and each barrel has it's own magazine, which makes it actually a two shotguns in a single frame.  Each barrel has it's own trigger, but because there is a single pump for both barrels, you can't really utilize it well, not saying the precision of the weapon do not make it fit for anything but shooting zombies at point blank.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "dp_12",
+        "name": { "str": "DP-12" },
+        "description": "Bullpup 12-gauge pump action double-barreled shotgun.  Each barrel has it's own magazine, which makes it actually a two shotguns in a single frame.  Each barrel has it's own trigger, but because there is a single pump for both barrels, you can't really utilize it well, not saying the precision of the weapon do not make it fit for anything but shooting zombies at point blank."
+      }
+    ],
+    "weight": "4989 g",
+    "volume": "8862 ml",
+    "//": "180 x 750 x 101 mm, trimmed about 35% down for empty spaces",
+    "longest_side": "749 mm",
+    "barrel_length": "479 mm",
+    "price": "1599 USD",
+    "price_postapoc": "35 USD",
+    "to_hit": { "grip": "solid", "length": "short", "surface": "any", "balance": "uneven" },
+    "material": [ "steel", "plastic" ],
+    "dispersion": 550,
+    "sight_dispersion": 700,
+    "durability": 7,
+    "valid_mod_locations": [
+      [ "barrel", 1 ],
```

Comment:
Shouldn’t there be 2 barrel and muzzle slots considering the weapon has two barrels? I’m not too familiar with how this gun works in function, so I could be wrong.

Reply:
it should be, yes, but i suspect it would not work as it should (meaning one shot uses gunmod 1, and another uses gunmod 2), so it would be used mostly for minmaxing - two silencers, for example